HW6/validators/body.schemas.ts

```diff
@@ -0,0 +1,17 @@
+import Joi from "joi";
+
+export const cartSchema = Joi.object({
+  productId: Joi.string().required(),
+  count: Joi.number().integer().min(0).required(),
```

Comment:
count with string value (i.e `"12"`) passes this validation

Reply:
Updated